wiki/Tournaments/oASIS/1/en.md

```diff
@@ -0,0 +1,198 @@
+  3. [xi - Beatrice (Gordon) \[Collab Insane\]](https://osu.ppy.sh/beatmapsets/1664961#osu/3477124)
+- Tiebreaker
+  1. **[wakaG - Yozora ni Saita Hana (Reillia) \[Sakura\]](https://osu.ppy.sh/beatmapsets/810188#osu/1699930)**
+
+### Qualifiers
+
+**[Download the mappack here! (67 MB)](https://drive.google.com/file/d/1zC0x2zbBZmUsKPypUvm7FE69u-708ton/view)**
+
+- NoMod
+  1. [yanaginagi - Zoetrope (Asphyxia) \[Erisu's Extra\]](https://osu.ppy.sh/beatmapsets/181681#osu/3711197)
+  2. [m108 - Sunflower (Leader) \[Expert\]](https://osu.ppy.sh/beatmapsets/1403962#osu/2895675)
+  3. [Takanashi Kiara - SPARKS (PotatoDew) \[don't stop!\]](https://osu.ppy.sh/beatmapsets/1693861#osu/3461183)
+  4. [Zekk - Feelsleft0ut (Ascended) \[Cubby's Extra\]](https://osu.ppy.sh/beatmapsets/1811658#osu/3716153)
+- Hidden
+  1. [BUTAOTOME - Sharekoube (moph) \[Expert\]](https://osu.ppy.sh/beatmapsets/1597784#osu/3894799)
+  2. [ICE - Majestic Phoenix (Zetera) \[Deramok's Lit Insane\]](https://osu.ppy.sh/beatmapsets/845094#osu/1772022)
+- HardRock
+  1. [BUTAOTOME - Kurukuru Tsukai (Entry) \[Elcheer's Another\]](https://osu.ppy.sh/beatmapsets/1032027#osu/2164424)
+  2. [Mitsukiyo - Luminous Memory (CoLouRed GlaZeE) \[Kuse's Insane\]](https://osu.ppy.sh/beatmapsets/1796025#osu/3681704)
+- DoubleTime
+  1. [Eri Sasaki - Ring of Fortune (CelsiusLK) \[Insane\]](https://osu.ppy.sh/beatmapsets/317236#osu/706798)
+  2. [SHK - SO HOT (Remix by SHK) (AngelHoney) \[Hard\]](https://osu.ppy.sh/beatmapsets/40086#osu/127409)
+  3. [HyuN - Illusion of Inflict (Icekalt) \[Light Insane\]](https://osu.ppy.sh/beatmapsets/968923#osu/2041171)
+
+## Ruleset
+
+### General rules
+
+1. Beatmap scoring is based on **[ScoreV2](/wiki/Gameplay/Score#scorev2).**
+2. The mappools for each round will be announced by the tournament management in advance before the actual matches take place.
+   - All beatmaps in the tournament will feature songs that are related to video games in one way or another.
+3. Match schedules will be predetermined by the tournament management. If there are any players who are unable to attend the current schedule for any reason, all other affected players may apply and settle for a reschedule at the `#scheduling` channel in the tournament's Discord server.
+4. A referee will create a multiplayer room 10 minutes in advance and will start to send out invites.
+5. If a player does not show up within **10 minutes** of the start time, their opponent wins by default.
+6. If no staff or referee is available, the match will be postponed.
+7. **NoFail will be enforced in all beatmaps.** This is to ensure that the points are to be awarded more fairly towards players who perform better in general during the course of the beatmap regardless of their remaining health at the end.
+8. If a player disconnects, the referee will note the score the player had achieved at the end of the beatmap directly from their PC and adjust the outcome of the pick accordingly.
```

Comment:
was this a lan tournament? that's not mentioned anywhere

Reply:
o woops i copied this elsewhere and forgot to adjust this accordingly lol